src/workerd/api/http.h

```diff
@@ -805,19 +805,16 @@ class Request final: public Body {
   Request(kj::HttpMethod method, kj::StringPtr url, Redirect redirect,
           jsg::Ref<Headers> headers, kj::Maybe<jsg::Ref<Fetcher>> fetcher,
           kj::Maybe<jsg::Ref<AbortSignal>> signal, CfProperty&& cf,
-          kj::Maybe<Body::ExtractedBody> body, CacheMode cacheMode = CacheMode::NONE)
+          kj::Maybe<Body::ExtractedBody> body, CacheMode cacheMode = CacheMode::NONE, kj::Maybe<jsg::Ref<AbortSignal>> thisSignal = kj::none)
```

Comment:
```suggestion
          kj::Maybe<Body::ExtractedBody> body, CacheMode cacheMode = CacheMode::NONE, 
          kj::Maybe<jsg::Ref<AbortSignal>> thisSignal = kj::none)
```

Reply:
I'm probably reading this wrong but that just looks like a clang-format thing?

Reply:
Yes. clang-format should have wrapped this line since it's over 100 chars long. But I see the lint check passed, so that's weird. Is clang-format failing to wrap here for some reason?

Reply:
I changed this line for other reasons and it's now under the length limit.